### PR TITLE
Write semver utility

### PR DESCRIPTION
**Pull request by afaq404alam**

### CODE REVIEW

File: src/utilities/cargo_utils.rs [hunk 1]
```diff
@@ -73,3 +78,50 @@ pub fn parse_config(doc: &Document, members: &Vec<String>) -> (String, Vec<Dep>)
...
+                    n + 1
+                } else {
+                    1
+                }
+            } else {
+                1
+            }
+        } else {
+            1
+        };
```
Comment (spacekookie) [inline src/utilities/cargo_utils.rs]: Can you please refactor this code to use `match` instead? It'll be shorter and easier to understand than three `else` that return 1's :wink:

**Review by spacekookie — COMMENTED** [review]
Thank you for this code!

It looks really good, for the most part. I commented on one chunk in the actual version bumping code I'd like to see changed before merging it.

File: src/utilities/cargo_utils.rs (hunk 1 above)
Comment (afaq404alam) [reply src/utilities/cargo_utils.rs]: Thanks for the review :) 
I started with `match` but then got caught up with `if let` 😬. I have updated the code.

Comment (spacekookie) [reply src/utilities/cargo_utils.rs]: `if let` is a slippery slope indeed! :joy:

**Review by spacekookie — APPROVED** [review]
Looks good, merging! :+1:

**Pull request merged**